_ml-commons-plugin/gpu-acceleration.md

```diff
@@ -0,0 +1,303 @@
+{: .warning}
+
+When running a natural language processing (NLP) model in your OpenSearch cluster with a machine learning (ML) node, you can achieve better performance on the ML node using graphics processing unit (GPU) acceleration. GPUs can work in tandem with the CPU of your cluster to speed up the model upload and training. 
+
+## Supported GPUs
+
+Currently, ML nodes support any GPU instances that use CUDA 11.6 or greater, including the following:
+
+- [NVIDIA G-series instances](https://aws.amazon.com/nvidia/)
+- [AWS Inferentia](https://aws.amazon.com/machine-learning/inferentia/)
+
+If you need GPU power, you can provision GPU instances through [Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/). For more information on how to provision a GPU instance, see [Recommended GPU Instances](https://docs.aws.amazon.com/dlami/latest/devguide/gpu.html).
+
+## Supported images
+
+You can use GPU acceleration with both [Docker images](https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/supported-tags.md) with CUDA 11.6 and [Amazon Machine Images (AMIs)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html).
+
+## PyTorch
+
+GPU-accelerated ML nodes require [PyTorch](https://pytorch.org/docs/stable/index.html) 1.12.1 or greater to work with ML models.
+
+## Setting up a GPU-accelerated ML node
+
+Depending on the GPU, you can provision a GPU-accelerated ML node manually or by using automated initialization scripts. 
+
+### Mounting an nvidia-uvm device
+
+NVIDIA uses CUDA to increase node performance. In order to take advantage of CUDA, you need to make sure that your drivers include the `nvidia-uvm` kernel inside the `/dev` directory. To check for the kernel, enter `ls -al /dev | grep nvidia-uvm`.
+
+If the `nvidia-uvm` kernel does not exist, run `nvidia-uvm-init.sh`:
+
+```
+#!/bin/bash
+## Script to initialize nvidia device nodes.
+## https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#runfile-verifications
+/sbin/modprobe nvidia
+if [ "$?" -eq 0 ]; then
+  # Count the number of NVIDIA controllers found.
+  NVDEVS=`lspci | grep -i NVIDIA`
+  N3D=`echo "$NVDEVS" | grep "3D controller" | wc -l`
+  NVGA=`echo "$NVDEVS" | grep "VGA compatible controller" | wc -l`
+  N=`expr $N3D + $NVGA - 1`
+  for i in `seq 0 $N`; do
+    mknod -m 666 /dev/nvidia$i c 195 $i
+  done
+  mknod -m 666 /dev/nvidiactl c 195 255
+else
+  exit 1
+fi
+/sbin/modprobe nvidia-uvm
+if [ "$?" -eq 0 ]; then
+  # Find out the major device number used by the nvidia-uvm driver
+  D=`grep nvidia-uvm /proc/devices | awk '{print $1}'`
+  mknod -m 666 /dev/nvidia-uvm c $D 0
+  mknod -m 666 /dev/nvidia-uvm-tools c $D 0
+else
+  exit 1
+fi
+```
+
+After verifying that `nvidia-uvm` exists under `/dev`, you can start OpenSearch inside your cluster. 
+
+### Preparing AWS Inferentia
```

Comment:
@Naarcha-AWS I see my comment closed https://github.com/opensearch-project/documentation-website/pull/2384#discussion_r1067356434, what do you think about changing to "Preparing AWS Inferentia ML node"?

> Prepare looks good, then maybe we should change to "Prepare AWS Inferentia ML node", because "AWS Inferentia" is not something we should prepare. For us, we concern the ML node environment rather than the GPU hardware itself.